src/deferred.js

```diff
@@ -37,7 +37,7 @@ function adoptValue( value, resolve, reject ) {
 	// For Promises/A+, convert exceptions into rejections
 	// Since jQuery.when doesn't unwrap thenables, we can skip the extra checks appearing in
 	// Deferred#then to conditionally suppress rejection.
-	} catch ( /*jshint -W002 */ value ) {
```

Comment:
What do you mean? The comment is aimed at a linter so the browsers don't matter.